Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG branch
 # Pin the base-url of the epel repo to ensure stability
 ADD misc/epel.repo /etc/yum.repos.d/epel.repo
 RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo >/etc/yum.repos.d/yarn.repo
-RUN yum install -y git gcc mongodb sudo git tar findutils make procps-ng python3 python3-devel git nodejs-grunt-cli gcc-c++ gcc make yarn postgresql
+RUN yum install -y git sudo git tar findutils make procps-ng python3 python3-devel git nodejs-grunt-cli gcc-c++ gcc make yarn postgresql
```

Comment:
Is git needed twice in this list?

Reply:
For GitOps! yes! /s
In all seriousness this dockerfile is a bit of a mess and fixing the whole thing is out of scope for this issue. (Although it should happen)